app/ai/voice/agents/breeze_buddy/utils/language_detector.py

```
@@ -0,0 +1,229 @@
+                        return language_code
+                    else:
+                        logger.warning(f"LLM returned unsupported language code: {language_code}")
+                        return None
+                else:
+                    response_text = await response.text()
+                    logger.warning(
+                        f"Gemini language detection failed: {response.status} - {response_text}"
+                    )
+                    return None
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)